src/ResourceManagement/ResourceManager/Authentication/MSITokenProvider.cs

```diff
@@ -18,13 +18,12 @@ namespace Microsoft.Azure.Management.ResourceManager.Fluent.Authentication
     /// </summary>
     public class MSITokenProvider : ITokenProvider, IBeta
     {
-        private readonly string resource;
+        private string resource;
         private readonly MSILoginInformation msiLoginInformation;
```

Comment:
jfyi: It seems we're initializing `this.msiLoginInformation` twice in the ctr (which I added by mistake when authored MSITokenProvider initially).